Bundles/Raven.Client.Authorization/Raven.Client.Authorization.xproj

```diff
@@ -9,7 +9,7 @@
     <ProjectGuid>43ebd466-4a7b-4cf3-85cd-72fecee22986</ProjectGuid>
     <RootNamespace>Raven.Client.Authorization</RootNamespace>
     <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">..\..\artifacts\obj\$(MSBuildProjectName)</BaseIntermediateOutputPath>
-    <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\bin\$(MSBuildProjectName)\</OutputPath>
+    <OutputPath Condition="'$(OutputPath)'=='' ">.\bin\</OutputPath>
```

Comment:
When using git you can stage per file or even per line - if your Visual Studio for some reason modify this you can simply skip those modifications in commit. 

For commit review and staging you can use for instance SourceTree: https://www.sourcetreeapp.com/